app/src/main/java/com/example/insights/HomeFragment.kt

```diff
@@ -28,7 +28,7 @@ class HomeFragment : Fragment() {
                     for (change in value.documentChanges) {
                         if (change.type == DocumentChange.Type.ADDED) {
                             videoValues.add(change.document.data as HashMap<String, String>)
-                            val Itemadapter = ItemAdapter2(this, videoValues)
+                            val VideoRecyclerViewAdapter = VideoRecyclerViewAdapter(this, videoValues)
                             recyclerView2.adapter = Itemadapter
```

Comment:
on the rightside is the value defined earlier, and this too should be updated

Reply:
done with the first change, but can you tell whats the second one?

Reply:
The adapter class name has been changes , and in this line we are setting the adapter for the given recylerview, so this line too should be changed.

Reply:
> The adapter class name has been changed , and in this line we are setting the adapter for the given recylerview, so this line too should be changed.

sir can you tell what exactly i have to change?
i have renamed val into video_recycler_view_adapter, in line 32 what exactly needs to be written in right side

Reply:
ok

Reply:
on line 32 the RHS should be video_recycler_view_adapter instead of Item adapter